v8/src/heap/base/asm/riscv/push_registers_asm.cc

```diff
@@ -90,4 +91,6 @@ asm(".global PushAllRegistersAndIterateStack             \n"
     "  lw s0, 0(sp)                                      \n"
     "  addi sp, sp, 56                                   \n"
     "  jr ra                                             \n");
+#else
+#error "RISC-V PushAllRegistersAndIterateStack unable to determine target arch."
```

Comment:
Patches to V8 should go into patches/, not be directly applied, otherwise the next update will blow them away.

But! - and this is important - changes should be submitted to upstream first, because I don't want to be forward-porting patches forever if I can avoid it. See https://v8.dev/docs/contribute for more info.